mygrad/math/consts.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+
+Inf = np.array(np.Inf)
```

Comment:
No need to cast these any of these as arrays, you can just make the alias:

```suggestion
Inf = np.Inf
```

MyGrad can work with float's just fine :)